build/tasks.ts

```diff
@@ -224,7 +224,7 @@ const ensureCloned = register("vscode:clone", async (runner) => {
 	}
 
 	runner.cwd = vscodePath;
-	const checkout = await runner.execute("git", ["checkout", "tags/1.31.1"]);
+	const checkout = await runner.execute("git", ["checkout", "tags/1.31.1"]); //TODO: this tag should come from a parameter
```

Comment:
What do you mean by a parameter?

Reply:
I guess he means it shouldn't be hard coded.